src/service/viewport/viewport-impl.js

```diff
@@ -782,6 +782,11 @@ export class Viewport {
   enterOverlayMode() {
     this.disableTouchZoom();
     this.disableScroll();
```

Comment:
This should be disabling scroll it sets overflow hidden but doesn't set the position to be fixed. Should move the code here.